tuf/api/metadata.py

```diff
@@ -251,27 +261,31 @@ def verify(self, key: Mapping[str, Any],
             A boolean indicating if the signature is valid for the passed key.
 
         """
-        signatures_for_keyid = list(filter(
-                lambda sig: sig.keyid == key['keyid'], self.signatures))
+        signatures_for_keyid = list(
+            filter(lambda sig: sig.keyid == key["keyid"], self.signatures)
+        )
 
         if not signatures_for_keyid:
-            raise tuf.exceptions.Error(
-                    f'no signature for key {key["keyid"]}.')
+            raise tuf.exceptions.Error(f'no signature for key {key["keyid"]}.')
 
         if len(signatures_for_keyid) > 1:
             raise tuf.exceptions.Error(
-                    f'{len(signatures_for_keyid)} signatures for key '
-                    f'{key["keyid"]}, not sure which one to verify.')
+                f"{len(signatures_for_keyid)} signatures for key "
+                f'{key["keyid"]}, not sure which one to verify.'
```

Comment:
Well spotted! I missed this when I prepared the PR. Looks like black doesn't change double quotes to single quotes nor adds escape characters, as a consequence it can't change the outer single quotes to double quotes. Unfortunately, pylint's quote consistency check also doesn't detect this, so I fear the onus will remain on the reviewer here. At least black does not change it back.

Reply:
Fixed in be0cef0.